File: VB/Sample/Models/Person.vb
Imports System
Imports System.ComponentModel.DataAnnotations

Namespace Sample.Models
    Public Class Person
        Public Sub New()
            PersonID = 0
            FirstName = String.Empty
            MiddleName = String.Empty
            LastName = String.Empty
            Roles = String.Empty
        End Sub

        Public Sub New(ByVal id As Integer, ByVal firstName As String, ByVal middleName As String, ByVal lastName As String, ByVal roles As String)
            Me.PersonID = id
            Me.FirstName = firstName
            Me.MiddleName = middleName
            Me.LastName = lastName
            Me.Roles = roles
        End Sub

        <Key> _
        Public Property PersonID() As Integer

        <Required(ErrorMessage := "First Name is required")> _
        Public Property FirstName() As String

        Public Property MiddleName() As String

        <Required(ErrorMessage := "Last Name is required")> _
        Public Property LastName() As String
        Public Property Roles() As Object
    End Class
End Namespace

File: CS/t328461/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace t328461.Models
{
    public class Person
    {
        public Person()
        {
            PersonID = 0;
            FirstName = string.Empty;
            MiddleName = string.Empty;
            LastName = string.Empty;
            Roles = string.Empty;
        }

        public Person(int id, String firstName, string middleName, String lastName, string roles)
        {
            this.PersonID = id;
            this.FirstName = firstName;
            this.MiddleName = middleName;
            this.LastName = lastName;
            this.Roles = roles;
        }

        [Key]
        public int PersonID { get; set; }

        [Required(ErrorMessage = "First Name is required")]
        public string FirstName { get; set; }

        public string MiddleName { get; set; }

        [Required(ErrorMessage = "Last Name is required")]
        public string LastName { get; set; }
        public object Roles { get; set; }
    }
}